cmd/admin-router.go

```diff
@@ -177,6 +186,15 @@ func registerAdminRouter(router *mux.Router, enableConfigOps bool) {
 		// Set Group Status
 		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/set-group-status").HandlerFunc(gz(httpTraceHdrs(adminAPI.SetGroupStatus))).Queries("group", "{group:.*}").Queries("status", "{status:.*}")
 
+		// Attach policies to group
+		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/attach-group-policies").HandlerFunc(gz(httpTraceHdrs(adminAPI.AttachPolicyToGroup))).Queries("group", "{group:.*}", "policiesToAdd", "{policiesToAdd:.*}")
```

Comment:
This should be POST, rather than a PUT. The reasoning is that PUT is expected to be idempotent - if the request is repeated, there will be no visible change in the response to the client, but in this case it is useful to actually return an error that the policy is already attached to the user.

We can also rename the parameter to `policiesToAdd` as just `policy` and allow it to be a repeated. We need to move away from comma separated policy lists, as comma is valid character in policy names per [AWS doc](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html) - we do not currently support this, but I'm hoping we can as we refactor. This means the policy query parameter has to be repeated like `?group=mygroup&policy=abc&policy=pqr` and must be URL encoded.

Reply:
I've added most of these changes but when the query is formatted as `policy=abc&policy=pqr`, mux.Vars only extracts the first appearance of `policy`. How should that be fixed?

Reply:
You can remove `policy` from the `Queries` call when setting up the routing. The path itself is enough. In the handler, you can find all the query parameters by accessing [`r.Form` as a map](https://pkg.go.dev/net/http#Request.FormValue).